pos_debranding/static/src/xml/Chrome.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates id="template" xml:space="preserve">
+
+    <t
+        t-name="Chrome"
+        t-inherit="point_of_sale.Chrome"
+        t-inherit-mode="extension"
+        owl="1"
+    >
+        <xpath expr="//img[hasclass('pos-logo')]" position="attributes">
+            <attribute name="style">display: none;</attribute>
```

Comment:
Seems to me this could've been done with css only, no?

```scss
img.pos-logo {
    display: none;
}
```

Anyway, not blocking. Nitpicking here ^^
Although with the css approach you'd probably save some cpu ticks, as it's one less qweb template to process.